Sources/PackagePlugin/PackagePlugin.docc/PackagePlugin.md

```diff
@@ -0,0 +1,223 @@
+# ``PackagePlugin``
+
+Create custom build steps and command-line actions for your Swift package.
+
+## Overview
+
+Use `PackagePlugin` to create plugins that extend Swift package manager's behavior in one of two ways:
+
+* term Build-tool plugins: Create a command that Swift package manager runs either before or during a target's build process.
```

Comment:
"runs either before or during a target's build process"

I'd try to word this more like "runs as a pre-build step before the entire build -OR- during the build of a specific target", since the options are:
- before the build entirely
- associated with a specific target

The current wording might imply that it can run "before a specific target", which isn't the case.